numpy/lib/tests/test_shape_base.py

```diff
@@ -238,6 +248,17 @@ class TestVsplit(TestCase):
     """Only testing for integer splits.
 
     """
+    def test_non_iterable(self):
+        assert_raises(ValueError, vsplit, 1,1)
+
+    def test_0D_array(self):
+        a = np.array(1)
+        try:
```

Comment:
This doesn't look right, what are you testing? Why catch the exception? Same below.